ansible/roles/test/files/acstests/acltb_test.py

```diff
@@ -0,0 +1,266 @@
+'''
+Description:    This file contains the ACL test for SONiC testbed
+
+                Implemented according to the https://github.com/Azure/SONiC/wiki/ACL-test-plan
+
+Usage:          Examples of how to use:
+                ptf --test-dir acstests acltb_test.AclTest  --platform remote -t 'router_mac="00:02:03:04:05:00";verbose=True;route_info="/tmp/route_info.txt"'
+'''
+
+#---------------------------------------------------------------------
+# Global imports
+#---------------------------------------------------------------------
+import random
+import time
+import logging
+import ptf.packet as scapy
+import socket
+import ptf.dataplane as dataplane
+
+from ptf.testutils import *
+from ptf.mask import Mask
+import ipaddress
+
+import os
+import logging
+import unittest
+
+import ptf
+from ptf.base_tests import BaseTest
+from ptf import config
+import ptf.dataplane as dataplane
+import ptf.testutils as testutils
+
+import pprint
+
+class AclTest(BaseTest):
+    '''
+    @summary: ACL tests on testbed topo: t1
+    '''
+
+    #---------------------------------------------------------------------
+    # Class variables
+    #---------------------------------------------------------------------
+    PORT_COUNT = 31 # temporary exclude the last port
+
+    def __init__(self):
+        '''
+        @summary: constructor
+        '''
+        BaseTest.__init__(self)
+        self.test_params = testutils.test_params_get()
+    #---------------------------------------------------------------------
+
+    def setUp(self):
+        '''
+        @summary: Setup for the test
+        '''
+
+        self.dataplane = ptf.dataplane_instance
+        self.router_mac = self.test_params['router_mac']
+
+    #---------------------------------------------------------------------
+
+    '''
+    For diagnostic purposes only
+    '''
+    def print_route_info(self):
+        pprint.pprint(self.route_info)
+        return
+    #---------------------------------------------------------------------
+
+    def verify_packet_any_port(self, pkt, ports=[], device_number=0):
```

Comment:
It is already there:
https://github.com/p4lang/ptf/blob/master/src/ptf/testutils.py#L2326
I borrowed this function from the fib_test.py; have no idea why the one from the ptf wasn't used...